docs/guides/component-testing/third-party-definitions.mdx

```diff
@@ -0,0 +1,245 @@
+# Integrating with Cypress Component Testing
+
+:::info
+This feature is only available from Cypress 12.7.
+:::
+
+
+Cypress ships official support for many popular libraries and frameworks, such as [React](/guides/component-testing/react/overview) and [Angular](/guides/component-testing/angular/overview). We constantly monitor what libraries developers are interested in and are always open to adding new ones to Cypress.
+
+All officially supported libraries feature a first class onboarding experience, where we detect and scaffold the correct files, and a framework-specific `mount` adapter, to render your components. We call this collection of features a *Framework Definition*, since it defines the requirements for a library or framework to work in Cypress.
```

Comment:
```suggestion
Cypress Component Testing includes [official support](/guides/component-testing/overview#Supported-Frameworks) for many popular libraries and frameworks such as [React](/guides/component-testing/react/overview), [Angular](/guides/component-testing/angular/overview), and [Vue](/guides/component-testing/vue/overview). All officially supported libraries feature a first class onboarding experience, where we detect and scaffold the correct files, and a framework-specific `mount` adapter to render your components. We call this collection of features a *Framework Definition* since it defines the requirements for a library or framework to work in Cypress.
```